app/decorators/exercise_decorator.rb

```diff
@@ -1,4 +1,5 @@
 class ExerciseDecorator < Draper::Decorator
+  EXERCISES_PATH = "#{Rails.root}/app/assets/exercises/"
```

Comment:
Did the documentation state that these view templates should go in the assets directory?